package.json

```diff
@@ -11,11 +11,7 @@
         "url": "git://github.com/lokalise/node-core.git"
     },
     "license": "Apache-2.0",
-    "files": [
-        "dist/**",
-        "LICENSE",
-        "README.md"
-    ],
+    "files": ["dist/**", "LICENSE", "README.md"],
```

Comment:
huh, I wonder why it is floating again

Reply:
Not sure ... the post version hook is there, let's see what happens after this release ...